docs/content/concepts/Formatting.js

```diff
@@ -112,6 +120,22 @@ const formats = {
             `position` defines ellipsis position and can be either `start`, `end` or `middle`. Default position is `end`.
         </Md></cx>
     },
+    "+": {
+        description: <cx><Md>
+            Displays a plus sign for positive numbers.
+        </Md></cx>
+    },
+    "c": {
+        key: true,
+        description: <cx><Md>
+            Used for compact number formatting, e.g. `105000` will be formatted as `105K`.
+        </Md></cx>
+    },
+    "a": {
+        description: <cx><Md>
+            Displays number in accounting format.
+        </Md></cx>
+    }
```

Comment:
These are additional flags for the number format. i.e. `n;0;2;c+` and should be listed under the number format, not as separate standalone formats.